app/rust/src/parser/id.rs

```diff
@@ -96,3 +111,29 @@ impl AssetId {
         self.0.to_bytes()
     }
 }
+
+#[derive(Clone, Debug, Default)]
```

Comment:
i notice there are a few derive(Debug)
that we can remove if compile for ledger targets:

```suggestion
#[derive(Clone, Default)]
  #[cfg_attr(any(feature = "derive-debug", test), derive(Debug))]
```